File: .tmp_py4013/batch_12_dump/3370_smallest_number_with_all_set_bits.js
// LeetCode 3370 - Smallest Number With All Set Bits
// https://leetcode.com/problems/smallest-number-with-all-set-bits/

var smallestNumber = function(n) {
    let x = 1;
    while (x < n) x = x * 2 + 1;
    return x;
};

File: .tmp_rs4013/rust_check16/3370_smallest_number_with_all_set_bits.rs
struct Solution;
// LeetCode 3370 - Smallest Number With All Set Bits
// https://leetcode.com/problems/smallest-number-with-all-set-bits/

impl Solution {
    pub fn smallest_number(n: i32) -> i32 {
        let mut x = 1;
        while x < n {
            x = x * 2 + 1;
        }
        x
    }
}

fn main() {}
